File: FizzBuzz.java
import java.util.Scanner;


public class FizzBuzz {
 public static void main(String a[]) {
  Scanner sc = new Scanner(System.in);

  int n = sc.nextInt();
  sc.close();

  for(int i = 1; i <= n; i++) {
   boolean three = (boolean) !((i % 3) != 0 );
   boolean five = (boolean) !((i % 5) != 0);
   if(three)
   	System.out.print("Fizz");
   if(five)
   	System.out.print("Buzz");
   if(!(three || five))
   	System.out.print(i);
   System.out.println();
   }
 }
}

File: FizzBuzz.rb
n = gets.chomp.to_i

n+=1
for i in 1..n
    three = ((i % 3) == 0)
    five = ((i % 5) == 0)
    if (three) then
        print "Fizz"
    end
    if (five) then
        print "Buzz"
    end
    if(not(three or five))
        print i
    end
    print "\n"
end
